projects/core/src/user/facade/user-address.service.ts

```diff
@@ -19,7 +19,7 @@ export class UserAddressService {
     authService: AuthService
   );
   /**
-   * @deprecated since version 1.x
+   * @deprecated since version 1.3
    *  Use constructor(store: Store<StateWithUser | StateWithProcess<void>>,
     authService: AuthService) instead
    */
```

Comment:
can we make `authService: AuthService) instead` to a new line, starts with `*`?

Reply:
Fixed.